integration-tests/cleanup.sh

```diff
@@ -14,13 +14,13 @@ fi
 if [ -d data_offline ]; then
     rm -rf data_offline
 fi
-if [ -f tx-query2-enclave-app.sgxs ]; then
-    rm tx-query2-enclave-app.sgxs
+if [ -L tx-query2-enclave-app.sgxs ]; then
```

Comment:
`-L` checks for symbolic link? why? isn't `-f` correct for the cleanup?

Reply:
yes, in my test, the file is a link and  always exist after the integration test, and may add into git if I use `git add .` command:
https://github.com/crypto-com/chain/blob/master/integration-tests/run.sh#L18

Reply:
It seems `-f` is indeed incorrect here 👍